### ISSUE TITLE
PHP Warning stops System

### ISSUE THREAD

**Issue by saschaluebeck** [tieupmedia/HwtTypo3HwtAddress#7]
Maybe I found another issue. I set up a new DEV system  (Typo3 9.5.9, hwt_address 0.2.1). Created a address list. When clicking an address to get the single view, my system stopped with a exception:

> (1/1) #1476107295 TYPO3\CMS\Core\Error\Exception
PHP Warning: Illegal offset type in isset or empty in C:\laragon\www\mydevpage\typo3\sysext\extbase\Classes\Persistence\Generic\PersistenceManager.php line 157

I was not able to prevent this warning. Only thing I could do: I added  ..

> 'SYS' => [
    'exceptionalErrors' => 4096,
],

.. to the LocalConfiguration to ignore the warning. Then I got my single view without any problem.

Not sure if this is a problem with my setup or the extension.

**Comment by tieupmedia** [comment]
Hi Sascha,

thanks again for reporting! 
Yes, I can confirm this bug and I'm going to fix it.

**Comment by tieupmedia** [comment]
@saschaluebeck Can you verify, if the current master at #628b159 solves this problem? If so, I'll directly release it as the next version (inluding the other fixes and improvements).

**Comment by saschaluebeck** [comment]
Actually I can't reproduce the bug. I removed the "exceptionalErrors" but the singleview still works. I deleted cache and temp Folder but the bug seems gone. Don't know. Still T3 9.5.9 and hwt_address 0.2.1. Maybe I must try a fresh DEV installation.

**Comment by saschaluebeck** [comment]
Nope, even with a fresh installation I can't reproduce this bug. Don't know what I did else.

But I bet you fixed the bug anyways :)

**Comment by tieupmedia** [comment]
Hey Sascha,

thanks for your testing effort! Have you now tested with the current master or the older version in which you have discovered this bug?

**Comment by saschaluebeck** [comment]
Sure thing!

I tested a fresh Typo3 9.5.9 with the hwt_address 0.2.1 from the repository. This does not include your fix but the bug seems gone. I can't remember what else I did to produce the bug.

> I set up a new DEV system (Typo3 9.5.9, hwt_address 0.2.1). Created a address list. When clicking an address to get the single view, my system stopped with a exception

This is exactly what I did but no error appears in the DEV where I got the bug before (I removed 'exceptionalErrors') neither in the fresh new DEV. Not sure if something changed on my apache or php conf or something.

So I can't really test if the bug is gone with the master as long as my regular version does not show the bug up either.
